package.json

```diff
@@ -36,17 +36,17 @@
     "cross-env": "^7.0.3",
     "esbuild": "^0.13.12",
     "eslint": "^8.3.0",
-    "eslint-define-config": "^1.1.4",
+    "eslint-define-config": "^1.2.0",
     "eslint-plugin-node": "^11.1.0",
     "execa": "^5.1.1",
     "fs-extra": "^10.0.0",
     "jest": "^27.3.1",
-    "lint-staged": "^12.1.1",
+    "lint-staged": "^12.1.2",
     "minimist": "^1.2.5",
     "node-fetch": "^2.6.6",
     "npm-run-all": "^4.1.5",
     "playwright-chromium": "^1.16.3",
-    "prettier": "2.4.1",
+    "prettier": "2.5.0",
```

Comment:
IMO we should wait a bit, before upgrading prettier. In the past there was always a x.1 or x.2 that fixed some issues reported by other users to prettier after they made some heavy decisions